sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseService.java

```diff
@@ -1752,6 +1752,8 @@ public void updateVaccinationStatuses(Long personId, Disease disease, Date vacci
 		// Only consider cases with relevance date at least one day after the vaccination date
 		if (vaccinationDate != null) {
 			vaccinationDate = DateHelper.getEndOfDay(vaccinationDate);
+		} else {
```

Comment:
Please turn this assignment around, i.e. check for null first and then return, so that the return statement is the first statement in the method.